tensorboard/main.py

```diff
@@ -114,6 +114,9 @@
     'event_file', '',
     'The particular event file to query for. Only used if --inspect is present '
     'and --logdir is not specified.')
+tf.flags.DEFINE_string(
+    'base_url', '',
```

Comment:
IMO, calling this `path_prefix` makes more sense to me. base_url seems to imply that this could be a full URL.

Reply:
Also, could you add a little more documentation here? ie, clarify that if provided and non-empty, this string should begin with "/"?

Reply:
Also, perhaps briefly note a use case for why one would change the `path_prefix`.

Reply:
@chihuahua thanks for your comments
I believe base_url was originally chosen as it is most similar with jupyter which also uses base_url similarly.

I'm not oppose to changing it as it does not indicate the entire base_url.

As per your last comment, are you suggesting adding a brief use case to the documentation or are you asking for clarification?

Reply:
@pbanavara do you have any reservations about changing it to path_prefix?

Reply:
Oh, I see. `base_url` sounds good then if there is precedence. I am suggesting maybe noting a use case. Why would someone want to set the `base_url`?

Reply:
@JGSweets no reservations from my side.

Reply:
While there's precedence, I do believe you are correct in that it may be confuse others. I like the suggestion of path_prefix and am inclined to keep it.

It may alter the base_url, it itself is not the base_url